jobs/release.Jenkinsfile

```diff
@@ -6,6 +6,9 @@ node {
     libcloud = load("libcloud.groovy")
 }
 
+def brew_principal = pipecfg.brew.principal
+def brew_profile = pipecfg.brew.profile
```

Comment:
what happens if these aren't defined? anything fail?

Reply:
Nothing fails, it will be defined as empty vars